lib/src/components/NxDropdownIconOnly/NxDropdownIconOnly.scss

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-button-helpers';
+@import '../../scss-shared/nx-dropdown-variables';
+
+.nx-dropdown--icon-only {
```

Comment:
Why are these modifiers on `.nx-dropdown` rather than this being a completely separate type of block?

Reply:
> Why are these modifiers on `.nx-dropdown` rather than this being a completely separate type of block?

I think that creating the modifier and changing a few values is easier than recreating all of the attributes for both the dropdown and the toggle